### ISSUE TITLE
Add example for header on static govReactable

### ISSUE THREAD

**Issue by sarahmwong** [dfe-analytical-services/shinyGovstyle#158]
## Overview
Missing examples for header on static govReactable, potentially need to think about how we link up the IDs too 

## How to reproduce
`
heading_text("Caption added separately", size = "l"),

govReactable(
  example_data,
  right_col = c("Bikes", "Vans", "Buses"),
  page_size = 5
)
`

## Additional context
Might be a similar fix to #141
